.eslintrc.json

```diff
@@ -1,11 +1,20 @@
 {
   "extends": [
+    "eslint:recommended",
```

Comment:
Updated this configuration. Each line does the following:
- `eslint:recommended`: follows the recommended rules [as defined in the eslint docs with the ✅ ](https://eslint.org/docs/latest/rules/)
- `plugin:@typescript-eslint/recommended`: follows the recommended rules [as defined in this rule config](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended.ts) and defined in the [general rules table](https://typescript-eslint.io/rules/?=recommended)
- `next/core-web-vitals`: includes NextJS base lint config plus two rules [as defined within this rules config](https://github.com/vercel/next.js/blob/3f907d38cbb3a97d5201d4add5757d64453756c7/packages/eslint-plugin-next/src/index.ts#L57)
- `prettier`: have prettier rules override conflicting eslint rules
- `plugin:storybook/recommended`: lint rules specific to storybook files